lib/js/components/Buttons/Button/Button.consts.ts

```diff
@@ -18,6 +18,14 @@ export const BUTTON_COLORS = {
 	INVERTED: 'inverted',
 } as const;
 
+export const BUTTON_COLORS_WITH_ICON_COLORS = {
```

Comment:
creating separate const can lead to misconsistencies. What'd you say about using `ICON_BUTTON_COLORS` to validate props?

Reply:
Thanks to the fixing import path:
```
import { BUTTON_TYPES } from '../Button/Button.consts';
```
this additional const is redundant. I always clicked suggestion by IDE to make short path in import and that makes errors.